examples/control-plane-metrics/output.yaml

```diff
@@ -42548,7 +42548,6 @@ metadata:
     app.kubernetes.io/version: "v0.37.3"
     app.kubernetes.io/managed-by: Helm
 spec:
-  minReadySeconds: 10
```

Comment:
This looks like a different change that is not part of this PR, Do I need to revert this?

Reply:
Hmm. yeah, that's really strange why that's being removed. That's also why the auto tests are failing below.

Reply:
Ah, the Grafana Agent has this in their code:
```
spec:
  {{- if ge (int .Capabilities.KubeVersion.Minor) 22 }}
  minReadySeconds: 10
  {{- end }}
```
Since we're doing a template, it's not checking the Kubernetes version of a cluster, but likely your kubectl?
Can you run `kubectl version` and `helm version` and paste the results?

Reply:
Kubectl
```
Client Version: v1.26.9
Kustomize Version: v4.5.7
Server Version: v1.27.6
```

Helm
```
version.BuildInfo{Version:"v3.11", GitCommit:"", GitTreeState:"", GoVersion:"go1.20.1"}
```

Reply:
That's so strange...
Can you edit the Makefile and change lines 25 and 26 to:
```Makefile
%/output.yaml: %/values.yaml
	helm template --kube-version 1.27.0 k8smon charts/k8s-monitoring -f $< > $@
```

Then re-run `make regenerate-example-outputs`. Check in the examples directory, but don't check-in the Makefile.

Reply:
Do I need to include this change in the Makefile?

Reply:
BTW that change works, now the `minReadySeconds: 10` is back

Reply:
Great. Let's not commit the Makefile change for now. I'll keep it in mind if I want to pin that version (and have one more thing to update occasionally).